### ISSUE TITLE
How to fix the training inputs same when running the 'mnist' example on gpu?

### ISSUE THREAD

**Issue by dianyancao** [pytorch/pytorch#2707]
I want to fixed all input the same when I run the code 'example/mnist/main.py' next time.
I fix np.random seed, torch cpu cuda random seed, and use shuffle=False for DataLoader,but the test result is different when I run the code next time with cuda,but normal on cpu,is there something to be set?

**Comment by soumith** [comment]
please use https://discuss.pytorch.org

**Comment by dianyancao** [comment]
Ok,thanks.
